File: test/fizzbuzztest.js
var assert = require("chai").assert;

var fizzbuzz = require("../app/fizzbuzz.js");

var expected = [];
for(var i=1; i<= 100; i++) {
  if(i % 15 == 0)       expected.push("FizzBuzz");
  else if (i % 5 == 0)  expected.push("Buzz");
  else if (i % 3 ==0)   expected.push("Fizz");
  else                  expected.push(i.toString());
}


describe("fizzbuzz", function() {

  it("1 to 100", function() {
    for(var i=1; i<= 100; i++) {
      assert.equal(fizzbuzz(i), expected[i-1]);
    }
  });
});

File: test/fizzbuzztest.rb
require_relative '../app/fizzbuzz'

expected = []
100.times do |i|
	if (i% 15 == 0)
		expected.push("FizzBuzz")
	elsif (i % 5 == 0)
		expected.push("Buzz")
	elsif (i % 3 == 0)
		expected.push("Fizz")
	else expected.push(i) 	
	end
end		

describe "fizzbuzz" do
	
	it "1 to 100" do
		100.times do |i|
			expect(fizzbuzz(i)).to eq(expected[i])
		end
	end	
end
